Advisor/Network/NetworkManager.swift

```diff
@@ -0,0 +1,44 @@
+//
+//  request.swift
+//  Advisor
+//
+//  Created by Ilgam Akhmatdionov on 30.09.2021.
+//
+
+import Foundation
+
+class NetworkManager {
+   
+    
+    func fetchData(completion: @escaping (Result<[Attraction], Error>) -> Void) {
+        
+        let urlString = "https://api.opentripmap.com/0.1/en/places/radius?radius=1000&lon=49.1233&lat=55.7879&format=json&apikey=5ae2e3f221c38a28845f05b6811ec1c8612b2b7b55d46425957df5a4"
+        guard let url = URL(string: urlString) else {return}
```

Comment:
it would be greate to provide custom error in callback as failure part of result it is is impossible to create url